DuckDuckGo/Updates/UpdateController.swift

```diff
@@ -168,14 +177,6 @@ final class UpdateController: NSObject, UpdateControllerProtocol {
             .assign(to: \.updateProgress, onWeaklyHeld: self)
 
         try updater?.start()
-
-#if DEBUG
-        updater?.automaticallyChecksForUpdates = false
-        updater?.automaticallyDownloadsUpdates = false
-        updater?.updateCheckInterval = 0
-#else
-        checkForUpdateIfNeeded()
```

Comment:
No longer necessary as `updater?.start()` already has `checkForUpdatesInBackground()` baked in. It will be called if we're due for an automatic update based on the configured 3 hour interval.

Reply:
Scratch this, we need it for the release note retrieval. For now at least.